staging/src/k8s.io/component-base/logs/config.go

```diff
@@ -62,30 +71,39 @@ func BindLoggingFlags(c *config.LoggingConfiguration, fs *pflag.FlagSet) {
 Runtime log sanitization may introduce significant computation overhead and therefore should not be enabled in production.`)
 }
 
-// UnsupportedLoggingFlags lists unsupported logging flags
-func UnsupportedLoggingFlags(normalizeFunc func(name string) string) []string {
-	allFlags := []string{}
+// UnsupportedLoggingFlags lists unsupported logging flags. The normalize
+// function is optional.
+func UnsupportedLoggingFlags(normalizeFunc func(f *pflag.FlagSet, name string) pflag.NormalizedName) []*pflag.Flag {
+	// TODO: verify --help output.
```

Comment:
Can we resolve this TODO before merging?

Reply:
It's resolved, checking `--help` output for undesired changes is part of the before/after compare script that I am running. Let me remove this comment.